client/src/components/TallyEntryUser/TallyEntryLoginScreen.tsx

```diff
@@ -0,0 +1,197 @@
+import React from 'react'
+import styled from 'styled-components'
+import {
+  Card,
+  FormGroup,
+  InputGroup,
+  HTMLSelect,
+  H1,
+  Button,
+  Colors,
+} from '@blueprintjs/core'
+import { useForm } from 'react-hook-form'
+import { useMutation, useQueryClient } from 'react-query'
+import { Inner, Wrapper } from '../Atoms/Wrapper'
+import { ITallyEntryUser, IMember } from '../UserContext'
+import { fetchApi } from '../../utils/api'
+
+const useRequestCode = () => {
+  const postRequestCode = async (body: { members: IMember[] }) =>
+    fetchApi(`/auth/tallyentry/code`, {
+      method: 'POST',
+      body: JSON.stringify(body),
+      headers: { 'Content-Type': 'application/json' },
+    })
+
+  const queryClient = useQueryClient()
+
+  return useMutation(postRequestCode, {
+    onSuccess: () => queryClient.invalidateQueries(['user']),
+  })
+}
+
+const MemberFieldset = styled.fieldset`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 15px;
+  text-align: left;
+  .bp3-form-helper-text {
```

Comment:
I historically haven't done this either, but we could use Blueprint's `Classes` object here instead of hardcoding, e.g.

```ts
const MemberFieldset = styled.fieldset`
  ...
  .${Classes.FORM_HELPER_TEXT} {
    height: 16px;
  }
`
```

Reply:
Sure, I like that